cmd/kube_create.go

```diff
@@ -537,7 +534,7 @@ func (c *sshClient) scp(src, dst string) error {
 		}
 	}
 
-	return ioutil.WriteFile(dst, buf.Bytes(), 0600)
+	return ioutil.WriteFile(dst, buf.Bytes(), 0o600)
```

Comment:
I think the "o" is not expected

Reply:
It is: https://www.infoworld.com/article/3408740/golang-113-adds-number-literal-prefixes.html

Reply:
my bad! :) thx